sapmachine-11/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM sapmachine:11
+
+ARG MAVEN_VERSION=3.8.4
+ARG USER_HOME_DIR="/root"
+ARG SHA=a9b2d825eacf2e771ed5d6b0e01398589ac1bfa4171f36154d1b5787879605507802f699da6f7cfc80732a5282fd31b28e4cd6052338cbef0fa1358b48a5e3c8
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN apt-get update \
+    && apt-get install -y curl git \
```

Comment:
git is not strictly required, the tests are there to catch regressions mostly, but up to you
https://github.com/carlossg/docker-maven/blob/master/tests/tests.bats#L63-L74

Reply:
We install git anyway in our custom version of this new upcoming image so I don't really mind, what would you prefer?